### PR TITLE
HIB-53 Some fixes and Docs updates

### PR DESCRIPTION
**Pull request by Askaholic**

### CODE REVIEW

File: cloudformation/tropo/templates/hyp3_autoscaling_group.py [hunk 1]
```diff
@@ -35,12 +41,24 @@
...
+from template import t
+from tropo_env import environment
+
 from .hyp3_keypairname_param import keyname
 from .hyp3_vpc import get_public_subnets, hyp3_vpc
 from .utils import get_map
 
 print('  adding auto scaling group')
 
+custom_metric_name = "RTCJobsPerInstance"
```
Comment (williamh890) [inline cloudformation/tropo/templates/hyp3_autoscaling_group.py:53]: Will this create a naming conflict with multiple stacks in the same region?

File: cloudformation/tropo/templates/hyp3_custom_metric.py [hunk 2]
```diff
@@ -97,9 +115,10 @@
     Arn=GetAtt(custom_metric, 'Arn'),
     Id="CustomMetricFunction1",
     Input=Sub(
-        '{"QueueUrl":"${QueueUrl}","AutoScalingGroupName":"${AGName}"}',
+        '{"QueueUrl":"${QueueUrl}","AutoScalingGroupName":"${AGName}","MetricName":"${MetricName}"}',
```
Comment (williamh890) [inline cloudformation/tropo/templates/hyp3_custom_metric.py:118]: line to long, from codefactor.

**Review by williamh890 — APPROVED** [review]

File: cloudformation/tropo/templates/hyp3_autoscaling_group.py (hunk 1 above)
Comment (Askaholic) [reply cloudformation/tropo/templates/hyp3_autoscaling_group.py:53]: No, the namespace is the stack name

File: cloudformation/tropo/templates/hyp3_custom_metric.py (hunk 2 above)
Comment (Askaholic) [reply cloudformation/tropo/templates/hyp3_custom_metric.py:118]: Yea by 1 character

**Pull request merged**